Source/ACE/Entity/WorldObject.cs

```diff
@@ -843,16 +843,16 @@ public virtual void SerializeCreateObject(BinaryWriter writer)
                 writer.Write((sbyte?)CombatUse ?? 0);
 
             if ((WeenieFlags & WeenieHeaderFlag.Structure) != 0)
-                writer.Write(Structure ?? 0u);
+                writer.Write(Structure ?? 0);
```

Comment:
Just plain "0" will default to a signed int.  You need to specify "ushort" in an explicit cast if that's what you need.